### ISSUE TITLE
Add support for Python 3.9

### ISSUE THREAD

**Issue by bmn001** [RMPR/atbswp#63]
## Checklist

- [x] I use the latest release of atbswp
- [x] The issue is not existing yet

## Verbose log
n/a

### Operating System
Ubuntu 21.04

### Desktop Environment/Window Manager
X11

### Python version
3.9.5

## Description

I used to run atbswp without a problem with Python 3.8.

When upgrading from Ubuntu 20.10 to 21.04, Python was updated to 3.9. Now atbswp won't launch anymore:

```
user@desktop:~/atbswp/atbswp$ python3 atbswp.py 
Traceback (most recent call last):
  File "/home/user/atbswp/atbswp/atbswp.py", line 21, in <module>
    import gui
  File "/home/user/atbswp/atbswp/gui.py", line 27, in <module>
    import control
  File "/home/user/atbswp/atbswp/control.py", line 30, in <module>
    import pyautogui
ModuleNotFoundError: No module named 'pyautogui'
```

**Comment by RMPR** [comment]
Hi, thanks for reporting, it seems like you need to follow the
install instructions once again. Pyautogui (and pynput) were installed
for the 3.8.

TL;DR: run ```python3 -m pip install pyautogui pynput --user```

**Comment by bmn001** [comment]
Thanks for the info!

I ran the four debian/ubuntu install lines again and now I'm back in business.
